proxy/src/graphql/schema.rs

```diff
@@ -359,6 +364,32 @@ pub struct ControlQuery;
 )]
 impl ControlQuery {}
 
+#[juniper::object]
+impl avatar::Avatar {
+    fn background(&self) -> avatar::Color {
+        self.background
+    }
+
+    fn emoji(&self) -> &str {
+        &self.emoji.0
+    }
+}
+
+#[juniper::object]
+impl avatar::Color {
+    fn r() -> i32 {
+        self.r as i32
+    }
+
+    fn g() -> i32 {
+        self.g as i32
+    }
+
+    fn b() -> i32 {
+        self.b as i32
+    }
+}
+
```

Comment:
Why do we need these? Why can't we just use the `u8` fields directly? Is this a juniper thing?

Reply:
Juniper doesn't implement their `Value` trait for any other primitive than `i32`.

Reply:
Ugh. Maybe for the frontend it would be easier to have a hex representation eventually, eg. `"#ff0011"`, because we won't need the individual channels. You could have eg.

```rust
impl fmt::Display for Color {
    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
        write!(f, "#{:02x}{:02x}{:02x}", self.r, self.g, self.b)
    }
}
```

Reply:
Thought about that as well, but the UI should determine how to handle colors. Esp. when there are plans to consistently move to `rgba()`. This would give the most flexibilty imho. @MeBrei Thoughts on this. Happy to hex, if it is the best format.